help/C/Help_ch_Transactions.xml

```diff
@@ -899,180 +905,543 @@ Author:
+          from the menu to open the QIF Import dialog box illustrated below.
         </para>
-      </sect3>
-      <sect3>
-        <title>Review the GnuCash accounts to be created.</title>
-        <para>The importer then matches up your QIF accounts and categories with GnuCash accounts and gives 
-              you a brief description of the matching process. Clicking Next will bring you to a view comparing
-              your QIF accounts with the corresponding GnuCash accounts created. To change an account name, select
-              the row containing that account name and edit the name in the dialog box provided. Click Next when 
-              you have finished making changes, and proceed through a similar category matching process. QIF 
-              income and expense categories import as GnuCash income and expense accounts <!--(see section 3.1 FIXME for 
-              more on this).--> Make changes to these account names if necessary, and click Next to continue.
-        </para>
-        <note><para>If you are not sure what changes are needed, it is safe to
-              accept the GnuCash account names. It is easy to edit the accounts later if you find you need to make
-              a change.
-        </para></note>
-        <para>From the drop-down list, select a standard currency to be used for the imported accounts and click <guibutton>Next</guibutton>
-              to continue. If you have stocks, mutual funds, or other securities, you will be prompted for additional
-              information. The importer dialog will ask for the exchange or listing (i.e. Nasdaq), the security's 
-              full name, and the ticker symbol. If you do not have this information handy, you can edit the account
-              information later, once the import is complete. Click <guibutton>Next</guibutton> button to continue.
-        </para>
-      </sect3>
-      <sect3>
-        <title>Tell GnuCash to import the data.</title>
-        <para>The last step is the import. Once you have verified your account names and investment information, 
-              click <guibutton>Finish</guibutton> button in the Update your GnuCash accounts page to complete the import process. Depending 
-              upon the size of your file, the import might take a few minutes to complete, so a progress bar 
-              displays the percentage finished. When the import process is complete, GnuCash will return you 
+        <screenshot id="ImportQIFDruid">
+        <mediaobject>
+          <imageobject role="html">
+            <imagedata fileref="figures/Help_Import_QIF_Druid.png" format="PNG" srccredit="David Cousens" width="510"></imagedata>
+          </imageobject>
+          <imageobject role="fo">
+                      <imagedata fileref="figures/Help_Import_QIF_Druid.png" format="PNG" srccredit="David Cousens"></imagedata>
+          </imageobject>
+          <textobject>
```

Comment:
Textobjects should be descriptions of the image, so that screenreaders can tell blind people something about the image. Repeating the title makes no sense. If you really want a text object fill it with the strings from the sources.

Reply:
Frank  I'll remove the text objects at least for the moment until i can conceive of a way to describe a screen to a blind person. and fix up the figures

Reply:
To get an impression of the text objects, you can open the html version with an textmode browser like links and it's derivates (lynx, elinks, ...):
`links https://code.gnucash.org/docs/C/gnucash-help/trans-import.html`